src/FileWatcher/index.js

```diff
@@ -52,9 +53,7 @@ class FileWatcher extends EventEmitter {
         );
       });
       return watchers;
-    });
-
-    this._loading.done();
+    }, console.error.bind(console));
```

Comment:
The constructor doesn't return the promise though. How else would an error be forwarded to somewhere the client could see?